Conversions/DateToDay.js

```diff
@@ -9,34 +9,34 @@
     algorithm shown below gives us the number of the day and
     finally converts it to the name of the day.
 
-    Algorithm & Explanation : https://cs.uwaterloo.ca/~alopez-o/math-faq/node73.html
+    Algorithm & Explanation : https://en.wikipedia.org/wiki/Zeller%27s_congruence
 */
 
 // March is taken as the first month of the year.
 const calcMonthList = {
-  1: 11,
-  2: 12,
-  3: 1,
-  4: 2,
-  5: 3,
-  6: 4,
-  7: 5,
-  8: 6,
-  9: 7,
-  10: 8,
-  11: 9,
-  12: 10
+  1: 13,
+  2: 14,
+  3: 3,
+  4: 4,
+  5: 5,
+  6: 6,
+  7: 7,
+  8: 8,
+  9: 9,
+  10: 10,
+  11: 11,
+  12: 12
 }
 
 // show the week day in a number : Sunday - Saturday => 0 - 6
 const daysNameList = { // weeks-day
```

Comment:
This should be an array.

Reply:
Done. Made it as an array, very good suggestion.